.github/workflows/api-docs_automation.yml

```diff
@@ -0,0 +1,50 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request 
+# events but only for the master branch
+on:
+  release:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+ 
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    # This will download new version of api-docs
+    - name : Update Api-docs
+      env:
+        token: ${{secrets.rdulmina_auth_tok}} 
```

Comment:
Shall we rename secret as something like "WEBSITE_TOKEN"

Reply:
Done